packages/backend/init-action/helpers/collaborators.js

```diff
@@ -8,6 +8,32 @@ const getCollaborators = async ({ octokit, owner, repo }) => {
       core.setFailed(error.message);
     });
 
+  const collaboratorInformation = projectCollaborators.map(collaborator =>
+    octokit
+      .request(`GET /users/${collaborator}`)
+      .then(c => c.data)
```

Comment:
I don't mind this sort of thing, but _maybe_ someone does?

`collaborator` maybe?